tests/e2e/testsLibrary/ProjectAndFileTests.ts

```diff
@@ -0,0 +1,33 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { CLASSES, Ide, ProjectTree, TestConstants, Editor } from '..';
+import { e2eContainer } from '../inversify.config';
+
+const ide: Ide = e2eContainer.get(CLASSES.Ide);
+const projectTree: ProjectTree = e2eContainer.get(CLASSES.ProjectTree);
+const namespace: string = TestConstants.TS_SELENIUM_USERNAME;
+const editor: Editor = e2eContainer.get(CLASSES.Editor);
+
+export function waitWorkspaceReadiness(workspaceName : string, sampleName : string, folder: string) {
+    test('Wait for workspace readiness', async () => {
+        await ide.waitWorkspaceAndIde(namespace, workspaceName);
+        await projectTree.openProjectTreeContainer();
+        await projectTree.waitProjectImported(sampleName, folder);
+    });
+}
+
+export function openFile(filePath: string, fileName: string) {
```

Comment:
Maybe ```openFileAndSelectEditorTab``` ?

Reply:
It is quite verbose I think.  If we would look at methods used on *.spec.ts as at BDD, then the person specifying this method would probably expect that if you open a file, its editor tab will be selected. So I would leave it as it is.

Reply:
I agree if change "selectTab" to "waitTabSelection" in other case it looks like additional step which doesn't reflected in the method name and may lead to the some misunderstanding